pkg/cmd/roachtest/testselector/snowflake_query.sql

```diff
@@ -14,14 +14,19 @@ with builds as (
 ), test_stats as (
   -- for all the build IDs select do a inner join on all the tests
   select test_name, -- distinct test names
-         count(case when duration>0 then test_name end) as total_runs, -- all runs with duration>0 (excluding all ignored test runs)
+         count(case when duration>0 and status='SUCCESS' then test_name end) as total_runs, -- all successful runs with duration>0
          -- count by status
          count(case when status='SUCCESS' then test_name end) as success_count,
-         count(case when status='FAILURE' then test_name end) as failure_count,
+         -- ignore the failures due to VM preempted VMs
```

Comment:
Nit: above comment is somewhat misleading since we're merely counting here, not filtering. Something like, "Count VM preemptions" is less ambiguous.